scripts/add-cahps-measures.js

```diff
@@ -0,0 +1,91 @@
+// Libraries
+const fs    = require('fs');
+const parse = require('csv-parse');
+const path  = require('path');
+
+const additionalMeasuresFilepath = '../util/additional-measures.json'
+var additionalMeasures = require(additionalMeasuresFilepath);
+
+// We want to overwrite CAHPS measures every time we run this script.
+additionalMeasures = additionalMeasures.filter(function(measure) {
+  var re = /CAHPS_\d+/i;
+  return measure.measureId.match(re) === null;
+});
+
+// Constants
+const CAHPS_CSV_COLUMNS = [
+  'Measure Name',
+  'Measure Points',
+  'Measure Contribution'
+];
+
+// Blurgh
+const defaultNqfId = '0005';
+const nqfIdMap = {
+  'CAHPS for MIPS SSM: Getting Timely Care, Appointments and Information': defaultNqfId,
+  'CAHPS for MIPS SSM: How Well Providers Communicate': defaultNqfId,
+  'CAHPS for MIPS SSM: Patient\'s Rating of Provider': defaultNqfId,
+  'CAHPS for MIPS SSM: Courteous and Helpful Office Staff': defaultNqfId
+};
+
+// Data
```

Comment:
Style comment nit - in general comments should explain why a code is functioning rather than what it is doing. In this case for example, the variable name is "data" so a comment marking it as data isn't needed.